app/views/admin/sponsors/_form.html.haml

```diff
@@ -8,12 +8,13 @@
         = @sponsor.name
 .row
   .col-md-8
-    = semantic_form_for(@sponsor, url: (@sponsor.new_record? ? admin_conference_sponsors_path : admin_conference_sponsor_path(@conference.short_title, @sponsor))) do |f|
+    = semantic_form_for(@sponsor, url: (admin_conference_sponsors_path)) do |f|
```

Comment:
If we only need this for creating a new sponsor, why not move this to new file instead of _form? It seems that a partial is no longer needed.